.github/workflows/vb_script.py

```diff
@@ -36,13 +36,15 @@ def bump_version(version_line, pre_release):
     curr_version = data[-1]
 
     if pre_release:
-        bumped_version = pl_version  # get current Pennylane version
-    else:
-        split_version = curr_version.split(".")  # "0.17.0" --> ["0,17,0"]
-        split_version[1] = str(int(split_version[1]) + 1)  # take middle value and cast as int and bump it by 1
+        curr_version = pl_version  # get current Pennylane version
```

Comment:
Just to make sure: this code was changed because previously it assumed that PennyLane would be released before we make the pre-release version bump of plugins?

Reply:
Yes the logic is such that it can handle the following cases: 
1. 
    - Pre-release version bump plugin
    - Release PL 
    - Release Plugin
    - Post-release version bump plugin 

2.
    - Pre-release version bump plugin 
    - Release plugin 
    - Post-release version bump plugin 
    - Release PL

I think there was a mis-communication before because this functionality was already present before. For some reason I thought you wanted to also include the following case: 

- Release PL 
- Pre-release version bump plugin 
- Release plugin 
- Post-release version bump plugin

Reply:
Oh, sorry about that! Sure, 1. sounds good.

Previously, the only other variant we've seen examples for was (e.g., now for Lightning):

* Pre-release version bump plugin
* Release PL
* Release Plugin
* Post-release version bump plugin